src/scripts.js

```diff
@@ -54,8 +58,8 @@ submitButton.addEventListener('click', () => {
 })
 
 // ~~~~~~~~~~~~~~ Setup Functions ~~~~~~~~~~~~~~~~~~~~
-function fetchData() {
-    Promise.all([getUserData, getRecipeData, getIngredientsData])
+function fetchData(urls) {
+    Promise.all([getData(urls[0]), getData(urls[1]), getData(urls[2])])
         .then(data => {
             apiUsers = data[0]
             apiRecipes = data[1]
```

Comment:
do we not need apiIngredients = data[2]?

Reply:
GitHub probs just doesn't show it, that's on line 66!